### PR TITLE
[RHCLOUD-46659] Kickoff Application Services Releases daily digest

### PR DESCRIPTION
**Pull request by g-duval**
[RHCLOUD-46659] Kickoff Application Services Releases daily digest, expected result looks like:
image

[RHCLOUD-46659]: https://redhat.atlassian.net/browse/RHCLOUD-46659?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Summary by CodeRabbit

* **New Features**
  * Added a daily Application Services digest email with per-product release sections, in-email navigation links, and a CTA to Subscriptions Inventory.
  * Aggregation now includes Application Services content and exposes a global releases counter.

* **Bug Fixes/Behavior**
  * Safer handling of missing/null fields during aggregation; products without releases are omitted and original ordering preserved.

* **Tests**
  * Added end-to-end and unit tests for aggregation, sorting, rendering, and multiple edge cases.

### CODE REVIEW

File: engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        action.setOrgId(DEFAULT_ORG_ID);
+
+        return action;
+    }
+
+    public static EmailAggregation createEmailAggregation(Action action, String eventTypeDisplayName) {
+        return TestHelpers.createEmailAggregationFromAction(action, eventTypeDisplayName);
+    }
+
+    public static Map<String, Object> buildApplicationServicesAggregatedPayload() {
```
Comment (bonscji1) [inline engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java]: this does not seems to be used anywhere?

File: engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+                .build()
+        ));
+
+        action.setAccountId(StringUtils.EMPTY);
+        action.setOrgId(DEFAULT_ORG_ID);
+
+        return action;
+    }
+
+    public static EmailAggregation createEmailAggregation(Action action, String eventTypeDisplayName) {
```
Comment (bonscji1) [inline engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java]: What is the point of this method?

File: engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java (hunk 1 above)
Comment (g-duval) [reply engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java]: nop, removed 😅, thanks

File: engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java (hunk 2 above)
Comment (g-duval) [reply engine/src/test/java/com/redhat/cloud/notifications/ApplicationServicesTestHelpers.java]: You are right it don't brings any value.
Removed.

**Review by bonscji1 — APPROVED** [review]
LGTM
